.eslintrc.js

```diff
@@ -23,17 +25,7 @@ module.exports = {
         specialLink: ['to'],
       },
     ],
-    'prettier/prettier': [
-      'warn',
-      {
-        printWidth: 100,
-        tabWidth: 2,
-        bracketSpacing: true,
-        trailingComma: 'es5',
-        singleQuote: true,
-        jsxBracketSameLine: false,
-      },
-    ],
+    'prettier/prettier': ['warn', prettierConfig],
```

Comment:
This is imported now, so that files that are non-JS can use the same prettier config.

Reply:
I am curious as to why eslint is configured to only warn about prettier. I imagine once a config for JS files is added to the lint-staged configuration, this would no longer be required?

Reply:
I think the initial logic was that we didn't want to discourage people from contributing because of it. But yeah, given the changes you have implemented here which help reinforce the standards, I think we can change it eventually.